apps/settings/elements/root.html

```diff
@@ -179,7 +179,7 @@ <h2 data-l10n-id="privacyAndSecurity">Privacy &amp; Security</h2>
         </li>
         <li class="privacy-panel-item" hidden>
           <a id="menuItem-privacyPanel" class="menu-item" data-icon="developer" href="#privacyPanel">
-            <span data-l10n-id="privacy-panel">Privacy Panel</span>
+            <span data-l10n-id="privacy-panel">Privacy Controls</span>
```

Comment:
Given the latest l10n guideline, let's remove the text from the element with an l10n id.